### DISCUSSION TITLE
Discord redirect url

### DISCUSSION THREAD

**Discussion by Simciutoja** [supabase/supabase#26695] [General]
it is possible to change url to sth else? Like for mine api files in react?

**Comment by silentworks** [comment]
Can you clarify what you are asking here? which URL are you asking if you can change? a screenshot would be helpful or some code example.

**Reply by Simciutoja** [reply]
URL from the discord redirect URL. Like to own link, example http://localhost/api/auth 

![image](https://github.com/supabase/supabase/assets/120786782/ac82d3aa-9fd6-48f7-b734-9a2929b77b58)
